backend/src/main/java/com/bakdata/conquery/models/query/results/ContainedEntityResult.java

```diff
@@ -22,4 +24,17 @@
 	default ContainedEntityResult asContained() {
 		return this;
 	}
+	
+	/**
+	 * Returns a list of the computed result line for this entity on the query.
+	 */
+	List<Object[]> listResultLines();
+	
+	/**
+	 * Allows to modify the underlying result directly. The return value of the line modifier is the new line.
+	 * So the modifier can change the array without reallocation by return the reference it received. Alternative it can 
+	 * allocate a new result line and return that reference instead. 
+	 * @param lineModifier A modifier(-chain) for a result line.
+	 */
+	void modifyResultLinesInplace(UnaryOperator<Object[]> lineModifier);
```

Comment:
```suggestion
	void applyModifiers(UnaryOperator<Object[]> lineModifier);
```

Aber brauchst du denn extra eine Funtkion, die ide Verkettung macht für dich?

Wenn ich das richtig verstehe löst du damit das Problem von

```
a = []
a.replaceAll(f)
a.replaceAll(g)
```

zu 

```
a.replaceAll(x -> g(f(x)))

```

?